# Relative two vertices
Vertex vertex_1 # id correapond to the sequence of Vertex msg
Vertex vertex_2

